frontend/components/gameBoxComponents/GameBoxArt.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Image, DefaultImage } from '../../storage/models/Image';
+import { fetchGameBoxArt } from '../../storage/utils/fetchGameData';
+const BoxArt = styled.img`
+	width: 320px;
+	margin-bottom: 20px;
+`;
+interface GameBoxArtState {
+	isDataAvailable: boolean;
+	image: Image | null;
+}
+
+interface GameBoxArtProps {
+	gameId: number;
+}
+
+export class GameBoxArt extends React.Component<GameBoxArtProps, GameBoxArtState> {
```

Comment:
Since our naming scheme is to use `Smart` for components which have logic (data fetching) in them it should be called `SmartGameBoxArt`